encrypted-media-respec.html

```diff
@@ -3209,10 +3209,46 @@
         </section>
       </section>
         
+      <section id="exposed-value-requirements">
+        <h3>Values Exposed to the Application</h3>
+        <p>Values exposed to the application could be used to identify the client or user, regardless of whether they are designed to be identifiers.
+          This section defines requirements for avoiding or at least mitigating such concerns.
+          There are additional requirements for <a href="#identifier-requirements">Identifiers</a>.
+        </p>
+
+        <section id="per-origin-per-profile-values">
+          <h3>Use Per-Origin Per-Profile Values</h3>
+          <!-- Issue #101 may affect this text. -->
+          <p>
+            All <a href="#distinctive-value">distinctive values</a> exposed to or inferable by the application MUST be unique per <a def-id="origin"></a> and <a def-id="browsing-profile"></a>.
+            That is, the value(s) used for one <a def-id="origin"></a> using the APIs defined in this specification MUST be different from those used for any other origin using the APIs,
+            and value(s) used in one <a def-id="browsing-profile"></a> MUST be different from those used for any other profile, regardless of origin.
+          </p>
```

Comment:
I think we can probable leave it, since there is not really a problem in practice, as you say.

My point is that all the issues related to distinctive values apply only if the value is provided to the application across multiple browsing sessions (allowing the application to say, "aha, I have seen this before, must be the same user"). A value could still be unique to a particular user, but if it local to a single browsing session, there's less of a tracking concern.

Now, with session ids, there is the possibility to query whether a given id is associated with a given user in a subsequent browsing session. This is a privacy issue only if the application already has another way to narrow down the set of possible users. If they have that, though, then an exhaustive search of possible session ids could reveal the user's identity.

Perhaps the definition of distinctive value should include that the value is visible across multiple browsing sessions ?